cfme/utils/appliance/implementations/ssui.py

```diff
@@ -105,6 +105,8 @@ def am_i_here(self):
             return self.view.is_displayed
         except (AttributeError, NoSuchElementException):
             return False
+        except NotImplementedError:
+            return True
```

Comment:
I think this check should actually go into the call to `self.am_i_here()` at L156/158 within `go`.

Then we can set `nav_args[wait_for_view] = 0` and stop it from waiting for the view - instead of defaulting a NotImplemented `is_displayed` to return True.